Lib/test/test_bisect.py

```diff
@@ -290,10 +282,7 @@ def test_arg_parsing(self):
             self.assertRaises(TypeError, f, 10)
 
 class TestErrorHandlingPython(TestErrorHandling, unittest.TestCase):
-    module = py_bisect
-
-class TestErrorHandlingC(TestErrorHandling, unittest.TestCase):
-    module = c_bisect
+    module = bisect
```

Comment:
Another option for this is to leave the `py_bisect = `/`c_bisect =`/`module = py_bisect` as is, and just add like `@unittest.skip("TODO: RUSTPYTHON, _bisect module")` to the C tests.

Reply:
Ah, yes. The reason for that is `test.support.import_fresh_module` isn't currently implemented. I could just change it to:

```python
...
import bisect
...

py_bisect = bisect
c_bisect = None
```

and then annotate the classes with `skip`. Adding skip is definitely a better option as it acts like a marker.